app/src/main/java/org/hackillinois/android/view/ProjectAdapter.kt

```diff
@@ -22,13 +22,19 @@ class ProjectAdapter internal constructor(private val mentorList: List<ProjectMo
         return viewHolder
     }
 
-    override fun onBindViewHolder(holder: ProjectAdapter.ViewHolder, position: Int) {
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
         val mentor = mentorList[position]
 
+        var mentors = ""
+        for (i in 0..mentor.mentors.lastIndex - 1) {
+            mentors += mentor.mentors[i] + ", "
+        }
+        mentors += mentor.mentors[mentor.mentors.lastIndex]
```

Comment:
This is a very confusing line. Why does a mentor have mentors? Maybe mentor is not a good name for this variable

Reply:
Is the mentor variable actually supposed to represent a project?

Reply:
Yeah this should be a project

Reply:
Yeah not sure what happened there whoops